pypopquiz/video.py

```diff
@@ -52,6 +51,15 @@ def filter_stream(stream: VideoBackend, kind: str, round_id: int, question: Dict
     return stream
 
 
+def get_backend(backend: str):
+    """Selects the backend based on a string name"""
+    if backend == 'ffmpeg':
+        return ppq.backends.ffmpeg.FFMpeg
+    elif backend == 'moviepy':
```

Comment:
pylint is unhappy here `pypopquiz/video.py:56:4: R1705: Unnecessary "elif" after "return" (no-else-return)`. It probably wants something like this:

```python
    if backend == 'ffmpeg':
        return ppq.backends.ffmpeg.FFMpeg
    if backend == 'moviepy':
        return ppq.backends.moviepy.Moviepy  # type: ignore
    raise ValueError('Invalid backend {} selected.'.format(backend))
```